src/lang/extra/czech.txt

```diff
@@ -572,81 +1106,1083 @@ STR_PROGSIG_INSERT                                              :Vložit
+
+STR_INDUSTRY_VIEW_NAME_TOOLTIP                                  :{STRING}
+STR_INDUSTRY_VIEW_REQUIRED_TOOLTIP_FIRST                        :{STRING}{STRING}
+STR_INDUSTRY_VIEW_REQUIRED_TOOLTIP_NEXT                         :, {STRING}{STRING}
+STR_INDUSTRY_VIEW_STOCKPILED_TOOLTIP                            :{CARGO_LONG} čeká{STRING}
+STR_INDUSTRY_VIEW_TRANSPORTED_TOOLTIP_EXTENSION                 :{}{BLACK}{CARGO_LONG} ({COMMA}%)
+
+STR_DEPOT_VIEW_COUNT_TOOLTIP                                    :{BLACK}Uvnitř je {COMMA} vozid{P lo la el}
+STR_DEPOT_VIEW_COUNT_STOPPED_TOOLTIP                            :{BLACK}Uvnitř je {COMMA} {P "zastavené vozidlo" "zastavená vozidla" "zastavených vozidel"}
+STR_DEPOT_VIEW_COUNT_WAITING_TOOLTIP                            :{BLACK}Uvnitř je {COMMA} {P "čekající vozidlo" "čekající vozidla" "čekajících vozidel"}
+STR_DEPOT_VIEW_TOTAL_TOOLTIP                                    :{BLACK}Uvnitř je {COMMA} vozidel:
+STR_DEPOT_VIEW_STOPPED_TOOLTIP                                  :{}{COMMA} {P je jsou jsou} v nečinném stavu
+STR_DEPOT_VIEW_WAITING_TOOLTIP                                  :{}{COMMA} {P čeká čekají čekají}
+STR_DEPOT_VIEW_FREE_WAGONS_TOOLTIP                              :{BLACK}Uvnitř je {COMMA} v{P ůz ozy ozů}
+STR_DEPOT_VIEW_MIXED_CONTENTS_TOOLTIP                           :{BLACK}{STRING}{}{STRING}
+
+STR_STATION_VIEW_NAME_TOOLTIP                                   :{STATION}{NBSP}{STATION_FEATURES}
+STR_STATION_VIEW_CARGO_LINE_TOOLTIP                             :{STRING} ({COMMA}%): {CARGO_SHORT}
+
+STR_VEHICLE_LIST_AGE                                            :{STRING}, stáří: {COMMA} {P rok roky let} ({COMMA})
+STR_VEHICLE_LIST_AGE_RED                                        :{STRING}, stáří: {RED}{COMMA} {BLACK}{P rok roky let} ({COMMA})
+STR_VEHICLE_LIST_CARGO_LIST                                     :{STRING}, náklad: {CARGO_LIST}
+STR_VEHICLE_LIST_RELIABILITY                                    :{STRING}, spolehlivost: {COMMA}%
+STR_VEHICLE_LIST_RELIABILITY_RED                                :{STRING}, spolehlivost: {RED}{COMMA}%
+STR_VEHICLE_LIST_MAX_SPEED                                      :{STRING}, max. rychlost: {VELOCITY}
+STR_VEHICLE_LIST_ENGINE_BUILT                                   :{STRING}, řada: {ENGINE}, rok výroby: {NUM}
+STR_VEHICLE_LIST_VALUE                                          :{STRING}, hodnota: {CURRENCY_LONG}
+STR_VEHICLE_LIST_LENGTH                                         :{STRING}, délka: {DECIMAL} políč{P ko ka ek}
+STR_VEHICLE_LIST_TIME_TO_LIVE                                   :{STRING}, životnost: {COMMA} {P rok roky let} zbývá
+STR_VEHICLE_LIST_TIME_TO_LIVE_RED                               :{STRING}, životnost: {RED}{COMMA} {BLACK}{P rok roky let} zbývá
+STR_VEHICLE_LIST_TIME_TO_LIVE_OVERDUE                           :{STRING}, životnost: {RED}{COMMA} {BLACK}year{P "" s} overdue
```

Comment:
Part of the text does not seem to be translated?

Reply:
Oh, yes, sorry. In addition, there was a wrong translation of the next string. Both fixed

Reply:
Sorry for that. Fixed